### ISSUE TITLE
Web support

### ISSUE THREAD

**Issue by babincc** [babincc/flutter_workshop#4]
This might not even be possible. Browsers are extremely restrictive with how a program communicates with the network.

**Comment by babincc** [comment]
I don't think this is possible in a realistic way. It would take a lot of hacky workarounds and third-party plugins

**Comment by Yayo-Arellano** [comment]
Web support should be added. The only restriction we can have is related to hue discovery using multicast DNS. But if he user adds the IP address manually there wont by any restriction.
